src/Sulu/Bundle/AdminBundle/Resources/js/components/MultiAutoComplete/MultiAutoComplete.js

```diff
@@ -41,8 +42,14 @@ class MultiAutoComplete extends React.Component<Props> {
 
     @observable labelRef: ElementRef<'label'>;
     @observable inputRef: ElementRef<'input'>;
+
+    @observable displaySuggestions = false;
```

Comment:
the suggestion list will be closed when clicking outside of the popover or when pressing `Escape`. it will be opened again when the search term in the input field is changed